### PR TITLE
Add NAC support for Powerconnect

### PR DESCRIPTION
**Pull request by vhuk**
Add NAC support for Dell N2048

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: mibs/dell/dellref.my
Comment (Jellyfrog) [inline mibs/dell/dellref.my:1]: Need to rename both mibs correctly. This should be named `DELL-REF-MIB`

Comment (vhuk) [reply mibs/dell/dellref.my:1]: Both MIBs have been renamed as requested.

**Review by Jellyfrog — APPROVED** [review]

File: LibreNMS/OS/Powerconnect.php [hunk 1]
```diff
@@ -112,4 +116,99 @@ public function pollProcessors(array $processors)
...
+            if ($char === '00') {
+                break;
+            }
+            $res .= chr(hexdec($char));
+        }
+
+        return $res;
+    }
+
+    private static function pivotTable(array $table): array
```
Comment (PipoCanaja) [inline LibreNMS/OS/Powerconnect.php:140]: Hi @vhuk 
Could you open a new PR to remove pivotTable here as well ? 
Thx

Comment (vhuk) [reply LibreNMS/OS/Powerconnect.php:140]: Done; opened PR #15796 for the change.

**Pull request merged**